lib/jwe/decrypt.js

```diff
@@ -223,12 +223,14 @@ function JWEDecrypter(ks, globalOpts) {
       promise = promise.then(function(jwe) {
         var processing = [];
         handlerKeys.forEach(function(h) {
-          h = extraHandlers[h];
+          var hv = extraHandlers[h];
```

Comment:
:wave:  why would you rename this to `hv` and if you do so why dont you give a more descriptive name to it?

Reply:
Because I tried to stay as close as possible to the original code and its descriptive naming conventions.